Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_CrescendoAnole.xml

```diff
@@ -33,22 +33,38 @@
 								<capacities>
 									<li>Scratch</li>
 								</capacities>
-								<power>10</power>
+								<power>5</power>
 								<cooldownTime>2.5</cooldownTime>
 								<linkedBodyPartsGroup>FrontLeftClaws</linkedBodyPartsGroup>
-								<armorPenetrationSharp>0.3</armorPenetrationSharp>
-								<armorPenetrationBlunt>1</armorPenetrationBlunt>
+								<surpriseAttack>
+									<extraMeleeDamages>
+										<li>
+											<def>Stun</def>
+											<amount>20</amount>
+										</li>
```

Comment:
Why did you move the stun here? 20 amount is bonkers even for a rough-plated monitor!

Reply:
Done(?)